auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -248,159 +260,185 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
+  /** Returns the type of the affiliation. */
+  protected type (business: Business): string {
+    if (this.isNameRequest(business)) {
+      return AffiliationTypes.NAME_REQUEST
+    }
+    if (this.isTemporaryBusiness(business)) {
+      return this.tempDescription(business)
     }
+    return GetCorpFullDescription(business.corpType.code as CorpTypeCd)
   }
 
-  /** Returns the temp corp description for display */
-  private tempDescription (corpType: CorpType): string {
-    switch (corpType) {
-      case CorpType.NEW_BUSINESS:
+  /** Returns the temp business description. */
+  protected tempDescription (business: Business): string {
+    switch (business.corpType.code as CorpTypes) {
+      case CorpTypes.INCORPORATION_APPLICATION:
         return AffiliationTypes.INCORPORATION_APPLICATION
-      case CorpType.NEW_REGISTRATION:
+      case CorpTypes.REGISTRATION:
         return AffiliationTypes.REGISTRATION
-    }
-  }
-
-  /** Returns the corp full description for display */
-  private typeDescription (corpType: CorpTypeCd): string {
-    return GetCorpFullDescription(corpType)
-  }
-
-  /** Returns the status for the affiliation */
-  private status (item: Business): string {
-    switch (true) {
-      case (this.isNameRequest(item.corpType.code) && !!item.nameRequest):
-        // Format name request state value for Display
-        if (!NrState[item.nameRequest?.state]) return 'Unknown'
-        return NrDisplayStates[NrState[item.nameRequest.state]] || 'Unknown'
-      case this.isTemporaryBusinessRegistration(item.corpType.code):
-        return BusinessState.DRAFT
-      case !!item.status:
-        return item.status.charAt(0)?.toUpperCase() + item.status?.slice(1)?.toLowerCase()
       default:
-        return BusinessState.ACTIVE
+        return '' // should never happen
     }
   }
 
-  /** Returns the folio number or a default message */
-  private folio (item: Business): string {
-    return item.nameRequest && (item.nameRequest.folioNumber || '')
-  }
-
-  /** Returns the last modified date string or a default message */
-  private lastModified (item: Business): string {
-    switch (true) {
-      case (!!item.lastModified):
-        return this.dateToPacificDate(new Date(item.lastModified))
-      case (!!item.modified):
-        return this.dateToPacificDate(new Date(item.modified))
-      default: return this.$t('notAvailable').toString()
+  /** Returns the type description. */
+  protected typeDescription (business: Business): string {
+    // if this is a name request then show legal type
+    if (this.isNameRequest(business)) {
+      const legalType: unknown = business.nameRequest.legalType
+      return GetCorpFullDescription(legalType as CorpTypeCd)
     }
+    // if this is an IA or registration then show legal type
+    if (this.isTemporaryBusiness(business)) {
+      const legalType: unknown = business.corpType.legalType
+      return GetCorpFullDescription(legalType as CorpTypeCd)
```

Comment:
At the moment, legal type isn't being provided, so this line returns ''.

I tested this by hard-coding a legal type -- works fine (see registration screenshot above).